epochStart/bootstrap/storageProcess.go

```diff
@@ -231,8 +231,9 @@ func (sesb *storageEpochStartBootstrap) createStorageRequesters() error {
 		return err
 	}
 
+	initialEpoch := uint32(1)
 	mesn := notifier.NewManualEpochStartNotifier()
-	mesn.NewEpoch(sesb.importDbConfig.ImportDBStartInEpoch + 1)
+	mesn.NewEpoch(initialEpoch)
```

Comment:
does this mean that you always start from genesis?
is it not possible to start from a later epoch?

Reply:
it can start from any epoch if the epoch data is contained in the db directrory. The `ImportDBStartInEpoch` flag was never used and it always carried 0 as value